cmake/platform.cmake

```diff
@@ -60,11 +60,13 @@ omr_add_prefix(OMR_ARCH_DEFINITIONS_PREFIXED ${OMR_C_DEFINITION_PREFIX} ${OMR_AR
 
 
 add_definitions(
-	${OMR_OS_COMPILE_OPTIONS}
 	${OMR_OS_DEFINITIONS_PREFIXED}
 	${OMR_ARCH_DEFINITIONS_PREFIXED}
 )
 
+set(CMAKE_C_FLAGS ${CMAKE_C_FLAGS} ${OMR_OS_COMPILE_OPTIONS})
+set(CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS} ${OMR_OS_COMPILE_OPTIONS})
```

Comment:
In my experience, no. However, I don't have a strong intuition for when stringification  is required in CMake.